app/src/main/java/com/github/gotify/messages/ListMessageAdapter.java

```diff
@@ -36,11 +39,7 @@
     private Settings settings;
     private Markwon markwon;
 
-    ListMessageAdapter(
-            Context context,
-            Settings settings,
-            Picasso picasso,
-            List<MessageWithImage> items,
+    ListMessageAdapter(Context context, Settings settings, Picasso picasso, List<MessageWithImage> items,
```

Comment:
Formatting is broken:
https://travis-ci.com/github/gotify/android/builds/181080210#L2278
Run `./gradlew spotlessApply`

Reply:
I never thought was failing because of formatting.